Tasks/AzureSpringCloudV0/azurespringclouddeployment.ts

```diff
@@ -2,7 +2,7 @@
 import tl = require('azure-pipelines-task-lib/task');
 import { TaskParameters, TaskParametersUtility } from './operations/taskparameters';
 
-import { AzureSpringCloudDeploymentProvider } from './deploymentProvider/AzureSpringCloudDeploymentProvider'
+import { AzureSpringAppsDeploymentProvider } from './deploymentProvider/AzureSpringCloudDeploymentProvider'
```

Comment:
Are we able to change "AzureSpringCloudDeploymentProvider"? Probably need to renam the file .

Reply:
Ok, will do, I was about to check with you whether this is needed.

Reply:
Thanks, good to know that.